Tic Tac Toe Game/index.js

```diff
@@ -0,0 +1,201 @@
+const boxes = document.querySelectorAll(".box");
+
+const isChecked = [false, false, false, false, false, false];
+let currentState = [0, 0, 0, 0, 0, 0, 0, 0, 0];
+
+let isGameOver = false;
+let winner = 0;
+let isUserOneTurn = true;
+
+boxes.forEach((box, index) => {
+  box.addEventListener("click", (e) => {
+    if (isChecked[index]) {
+      console.log("returning at ", index);
+      return;
+    }
+
+    // console.log(e.target);
+    // console.log(e.currentTarget);
+
+    isChecked[index] = true;
+    e.target.classList.add("visible");
+
+    if (isUserOneTurn) {
+      box.querySelector("i").classList.add("fa-circle");
+      box.querySelector("i").classList.add("fa-regular");
+      currentState[index] = 1;
+    } else {
+      box.querySelector("i").classList.add("fa-xmark");
+      box.querySelector("i").classList.add("fa-solid");
+      currentState[index] = -1;
+    }
+    isUserOneTurn = !isUserOneTurn;
+    // console.log(currentState);
+
+    checkGameOver();
```

Comment:
instead of using global isGameOver variable return it from checkGameOver function